lib/pbkit/pbkit.c

```diff
@@ -2287,183 +2287,126 @@ void pb_end(DWORD *pEnd)
             }
         }
     }
+    return NULL;
 }
 
-
-
-void pb_push1to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1)
+inline void pb_pushto(DWORD subchannel, DWORD *p, DWORD command, DWORD nparam)
```

Comment:
These should probably still return the next valid `p`?

Reply:
Should be fixed now.